mylyn.tasks/org.eclipse.mylyn.tasks.ui/src/org/eclipse/mylyn/internal/tasks/ui/editors/TaskEditorCommentPart.java

```diff
@@ -82,19 +82,19 @@ public class TaskEditorCommentPart extends AbstractTaskEditorPart {
 
 	public class CommentGroupViewer {
```

Comment:
Such a massive increase of fields and methods visibility indicates an issue in design. But we can postpone this discussion.

Reply:
Yes, should I create a Issue for this?

Reply:
Yes, why not

Reply:
Now we have Issue #266